source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -153,16 +154,19 @@ export default class NetworkStatusStore extends Store {
     );
   };
 
-  @action async _restartNode() {
+  _restartNode = async () => {
```

Comment:
@daniloprates why did we need to do this change? Can't this method remain an action?

Reply:
The main purpose of this function is to restart node, which doesn't require it to be an action, as we call `await restartCardanoNodeChannel.send();`. 

I'm not entirely sure why, but as an action it loses the Class instance and `this` becomes a global object, so `this.isNodeRestarting = true` doesn't work. 

With these changes, the main function is not an action, as it doesn't need to be, and `this.isNodeRestarting = true;` is wrapped into a `runInAction` method.

Reply:
OK thanks for the answer @daniloprates